client/app/components/Resource/__tests__/Resource.spec.jsx

```diff
@@ -17,68 +17,49 @@ const AUTHOR = 'Desi Rottman';
 const URL = 'http://www.lifesigns.org.uk/';
 
 describe('Resource', () => {
-  it('renders with neither tags nor author', () => {
-    let wrapper = shallow(<Resource title={TITLE} link={URL} />);
-    expect(wrapper.find('.author').exists()).toEqual(false);
-    expect(wrapper.find('.tags').exists()).toEqual(false);
-
-    wrapper = shallow(<Resource title={TITLE} link={URL} tags={TAGS} />);
-    expect(wrapper.find('.author').exists()).toEqual(false);
-    expect(wrapper.find('.tags').exists()).toEqual(false);
-
-    wrapper = shallow(<Resource title={TITLE} link={URL} author={AUTHOR} />);
-    expect(wrapper.find('.author').exists()).toEqual(false);
-    expect(wrapper.find('.tags').exists()).toEqual(false);
-
-    wrapper = shallow(
-      <Resource title={TITLE} link={URL} tags={TAGS} author={AUTHOR} />,
-    );
-    expect(wrapper.find('.author').exists()).toEqual(false);
-    expect(wrapper.find('.tags').exists()).toEqual(false);
+  const { getByRole, getByText } = screen;
+  test('it renders with neither tags nor author', () => {
+    render(<Resource title={TITLE} link={URL} />);
+    expect(getByText(TITLE)).toBeInTheDocument();
+    expect(getByRole('link')).toBeInTheDocument();
   });
-
-  it('renders with only tags', () => {
-    let wrapper = shallow(
-      <Resource title={TITLE} link={URL} tagged tags={TAGS} />,
-    );
-    expect(wrapper.find('.author').exists()).toEqual(false);
-    expect(wrapper.find('.tags').exists()).toEqual(true);
-    expect(wrapper.find('Tag').length).toEqual(TAGS.length);
-
-    wrapper = shallow(
-      <Resource title={TITLE} link={URL} tagged tags={EMPTY_TAGS} />,
-    );
-    expect(wrapper.find('.author').exists()).toEqual(false);
-    expect(wrapper.find('.tags').exists()).toEqual(true);
-    expect(wrapper.find('Tag').length).toEqual(0);
+  test('it renders with tags={TAGS} and no tagged prop', () => {
+    render(<Resource title={TITLE} link={URL} tags={TAGS} />);
```

Comment:
Taking a look at the Resources component it seems like tagged is a redundant prop. If `tags` is undefined then that should be enough to tell us that there are no tags. We don't need `tagged`.

I would refactor the [component](https://github.com/ifmeorg/ifme/blob/master/client/app/components/Resource/index.jsx#L12) to simplify this logic and then update this test.

Reply:
I'm going to get started on all of this tonight :) I'll reach out and let you know if I have any questions !